doc/varnish-configuration.rst

```diff
@@ -88,23 +107,22 @@ Varnish contains a `ban lurker`_ that crawls the content to eventually throw out
 Tagging
 ~~~~~~~
 
-Add the following to your Varnish configuration to enable :ref:`cache tagging <tags>`.
+If you have included fos_ban.vcl, tagging will be automatically enabled using ``X-Cache-Tags`` header :ref:`cache tagging <tags>`.
 
 .. note::
-
-    The custom ``X-Cache-Tags`` header should match the tagging header
-    :ref:`configured in the cache invalidator <custom_tags_header>`.
+    If you want to use different tag from ``X-Cache-Tags`` as defined in fos_ban.vcl,
+    the header should match the tagging header :ref:`configured in the cache invalidator <custom_tags_header>`.
```

Comment:
i think this is a bit confusing. i would say something like:

If you need to use a different tag for the headers than the default ``X-Cache-Tags`` used in ``fos_ban.vcl``,
you need to write your own VCL code and change the tagging header :ref:`configured in the cache invalidator <custom_tags_header>`.